posthog/api/team.py

```diff
@@ -135,19 +135,23 @@ def validate(self, attrs: Any) -> Any:
         return super().validate(attrs)
 
     def create(self, validated_data: Dict[str, Any], **kwargs) -> Team:
-        try:
-            serializers.raise_errors_on_nested_writes("create", self, validated_data)
-            request = self.context["request"]
-            organization = self.context["view"].organization  # Use the org we used to validate permissions
-            with transaction.atomic():
-                team = Team.objects.create_with_data(**validated_data, organization=organization)
-                request.user.current_team = team
-                request.user.save()
-                if validated_data.get("is_demo", False):
-                    MatrixManager(HedgeboxMatrix(), use_pre_save=True).run_on_team(team, request.user)
-        except Exception as e:  # TODO: Remove this after 2022-12-07, the except is just temporary for debugging
-            capture_exception()
-            raise e
+        serializers.raise_errors_on_nested_writes("create", self, validated_data)
+        request = self.context["request"]
+        organization = self.context["view"].organization  # Use the org we used to validate permissions
+        with transaction.atomic():
+            try:
+                with transaction.atomic():
+                    team = Team.objects.create_with_data(**validated_data, organization=organization)
+                    request.user.current_team = team
+                    request.user.save()
+                    if validated_data.get("is_demo", False):
+                        MatrixManager(HedgeboxMatrix(), use_pre_save=True).run_on_team(team, request.user)
+            except DatabaseError as db_err:
```

Comment:
this bit doesn't really do anything, is a duplicate of below?

Reply:
Yep, `DatabaseError` inherits from base `Exception`, so the latter catches the former as well